docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -1069,25 +1069,36 @@ function uiowa_core_token_info(): array {
 function uiowa_core_tokens($type, $tokens, array $data, array $options, BubbleableMetadata $bubbleable_metadata) {
   $replacements = [];
   if ($type === 'uiowa_core') {
-
     $config_factory = \Drupal::configFactory();
     $config = $config_factory->getEditable('uiowa_core.settings');
     $theme_config = $config_factory->getEditable('uids_base.settings');
     $ccom_site = ($theme_config->get('header.branding_options') == 'ccom');
 
     foreach ($tokens as $name => $original) {
+
       // Find the desired token by name.
       switch ($name) {
-        case 'org_name':
-          $org_name = $ccom_site ? 'Carver College of Medicine' : 'The University of Iowa';
-          $replacements[$original] = $config->get('uiowa_core.org_name') ?? $org_name;
-          break;
-
         case 'favicon_assets_path':
           $favicon_path = $ccom_site ? '/profiles/custom/sitenow/assets/uihc_favicons/' : '/profiles/custom/sitenow/assets/';
           $replacements[$original] = $config->get('uiowa_core.favicon_assets_path') ?? $favicon_path;
 
           break;
+
+        case 'combined_org':
+          $site_settings = $config_factory->getEditable('system.site');
+          $formatted_org = 'The University of Iowa';
```

Comment:
We still need the `$config->get('uiowa_core.org_name')` check for the [regents option](https://iowa.sharepoint.com/sites/drupal-developer-documentation/SitePages/SiteNow%20Drupal%209/Notes%20for%20regent%20option%20websites/Notes%20for%20regent%20option%20websites.aspx).

```suggestion
          $formatted_org = $config->get('uiowa_core.org_name') ?? 'The University of Iowa';
```